### DISCUSSION TITLE
Access for visitors

### DISCUSSION THREAD

**Discussion by perhp63** [microsoft-search/pnp-modern-search#4003] [Q&A]
Probably not a bug, but Ican't see what is wrong. None of the PnP parts show up, unless I log in as an administrator. The desciption of the error message says "object object". It seems as if have to allow my users to get access, but I can't fint the information om how to do this. 

The full error message is:

===========================

Something went wrong
If the problem persists, contact the site administrator and enter the information in Technical Information.
Technical information
WRONG:
[object Object]

===========================

Can someone help me?

BTW: A great tool 😊

<img width="606" alt="PnP" src="https://github.com/user-attachments/assets/c4829db6-4d03-470c-8b64-b74564ae2e00">

**Comment by kasperbolarsen** [comment]
HI :-) and thanks,
Can you check that the package in installed correctly in the app store:
https://[YourDomain].sharepoint.com/sites/appcatalog/_layouts/15/tenantAppCatalog.aspx/manageApps
It should look like this:
![image](https://github.com/user-attachments/assets/f6ddb7e0-0349-4a5e-ae42-487702dfeab5)

**Comment by perhp63** [comment]
Hi Kaspar and thanks for the quick reply

I installed PnP by downloading the file pnp-modern-search-parts-v4.sppkg

This is how my Tenant look, compared to yours:

***@***.***



Från: Kasper Larsen ***@***.***>
Skickat: den 23 september 2024 09:15
Till: microsoft-search/pnp-modern-search ***@***.***>
Kopia: perhp63 ***@***.***>; Author ***@***.***>
Ämne: Re: [microsoft-search/pnp-modern-search] Access for visitors (Issue #4000)


HI :-) and thanks,
Can you check that the package in installed correctly in the app store:
https://[YourDomain].sharepoint.com/sites/appcatalog/_layouts/15/tenantAppCatalog.aspx/manageApps
It should look like this:
image.png (view on web)<https://github.com/user-attachments/assets/f6ddb7e0-0349-4a5e-ae42-487702dfeab5>

-

**Comment by kasperbolarsen** [comment]
@perhp63  The picture was stripped :-(

**Comment by perhp63** [comment]
Ok, Here is a screendump




Från: Kasper Larsen ***@***.***>
Skickat: den 23 september 2024 11:32
Till: microsoft-search/pnp-modern-search ***@***.***>
Kopia: perhp63 ***@***.***>; Mention ***@***.***>
Ämne: Re: [microsoft-search/pnp-modern-search] Access for visitors (Issue #4000)


@perhp63<https://github.com/perhp63> The picture was stripped :-(

-

**Comment by kasperbolarsen** [comment]
email seems to remove any images

**Comment by perhp63** [comment]
Hmm

Ok, here is a link to the picture: https://struktiv.se/temp/PnP2.png

[En bild som visar Människoansikte, klädsel, person, person  Automatiskt genererad beskrivning]

Kvalitets- och skolutvecklare

Per Hansson

***@***.******@***.***>   |   076-199 20 80   |   struktiv.se<http://www.struktiv.se/>   |   Spånga, Stockholm
[En bild som visar text, skärmbild, Electric blue, Teckensnitt  Automatiskt genererad beskrivning]


Från: Kasper Larsen ***@***.***>
Skickat: den 23 september 2024 12:08
Till: microsoft-search/pnp-modern-search ***@***.***>
Kopia: perhp63 ***@***.***>; Mention ***@***.***>
Ämne: Re: [microsoft-search/pnp-modern-search] Access for visitors (Issue #4000)


email seems to remove any images

-

**Comment by kasperbolarsen** [comment]
Oh no, I know that one too well. I made a fumle in the DK language pack and that caused that error. Upgrade to the 4.13.2 version and you should be good .

**Comment by wobba** [comment]
Take a look at this awesome post and my guess is you will find your solution https://laurakokkarinen.com/sharepoint-online-guest-user-troubles-and-how-to-get-past-them/

**Comment by perhp63** [comment]
Problem solved 😊.

We used Lauras CDN-solution and it worked like a Charm.
